.azure-pipelines-templates/test.yml

```diff
@@ -10,6 +10,22 @@ steps:
   displayName: CTest
   workingDirectory: build
 
+# If a test fails, crudely upload nodes' logs for all tests
```

Comment:
It would also be useful to copy the output of ctest

Reply:
That goes to the UI via publish test results and xslt.